app/src/main/java/net/skyscanner/backpack/demo/compose/ButtonsStory.kt

```diff
@@ -45,39 +45,107 @@ import net.skyscanner.backpack.compose.icons.lg.LongArrowRight
 import net.skyscanner.backpack.compose.icons.sm.LongArrowRight
 import net.skyscanner.backpack.compose.tokens.BpkSpacing
 import net.skyscanner.backpack.demo.R
-import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
 import kotlin.time.ExperimentalTime
 
 @Composable
 fun ButtonsStory(
   size: BpkButtonSize,
   modifier: Modifier = Modifier,
 ) {
-  LazyColumn(modifier.fillMaxSize()) {
-    item {
-      ButtonsRow(
-        type = BpkButtonType.Primary,
+
+  val icon = when (size) {
+    BpkButtonSize.Default -> BpkIcons.Sm.LongArrowRight
+    BpkButtonSize.Large -> BpkIcons.Lg.LongArrowRight
+  }
+
+  Row(
+    modifier = modifier
+      .fillMaxSize()
+      .padding(horizontal = BpkSpacing.Base)
+      .padding(top = BpkSpacing.Md),
+    verticalAlignment = Alignment.Top,
+    horizontalArrangement = Arrangement.SpaceBetween,
+  ) {
+
+    ButtonsColumn { type, enabled ->
```

Comment:
Columns are needed to align all the buttons in the same way even if its width are different – it's relevant now since button link has different width